app/src/main/res/layout/activity_setting.xml

```diff
@@ -40,5 +40,44 @@
         android:layout_below="@+id/label_take_me_home"
         android:layout_alignStart="@+id/label_take_me_home" />
 
+    <TextView
+            android:text="@string/label_transportations"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/input_take_me_home"
+            android:id="@+id/label_transportations"
+            android:labelFor="@+id/label_transportations"
```

Comment:
labelFor should reference the id of the LinearLayout with the Checkboxes and not itself.